### PR TITLE
Feature/updated scss

### PR DESCRIPTION
**Pull request by PriyaVeruva**

### CODE REVIEW

File: src/components/common/Card/Card.scss [hunk 1]
```diff
@@ -24,15 +21,15 @@ $font-family-2: "Futura-Medium";
 }
 
 .subHeading {
-	font-family: $font-family-2;
+	// font-family: $font-family-2;
```
Comment (ShakirHajabba) [inline src/components/common/Card/Card.scss:24]: @PriyaVeruva remove comments.

File: src/components/common/Card/Card.scss [hunk 2]
```diff
@@ -211,15 +212,9 @@ $font-family-2: "Futura-Medium";
 	border-radius: 4px;
 	margin-block: 40px;
 	font-size: 18px;
-	font-family: $font-family-2;
+	// font-family: $font-family-2;
```
Comment (ShakirHajabba) [inline src/components/common/Card/Card.scss:215]: @PriyaVeruva remove all comments in the file.

File: src/components/common/CarouselComponent/Carousel.component.tsx [hunk 3]
```diff
@@ -1,122 +1,149 @@
...
-    singleCarousel: boolean;
-    carouselStyles: boolean;
+	carouselData: carouselType[];
+	singleCarousel: boolean;
+	carouselStyles: boolean;
 };
 type arrowDataType = {
-    onClick?: () => void;
+	onClick?: () => void;
 };
```
Comment (ShakirHajabba) [inline src/components/common/CarouselComponent/Carousel.component.tsx:21]: @PriyaVeruva Move all types to .type file.

File: src/components/common/CarouselComponent/Carousel.component.tsx [hunk 4]
```diff
@@ -1,122 +1,149 @@
...
-              ],
-    };
+function CarouselComponent({
+	carouselData,
+	singleCarousel,
+	carouselStyles,
+}: carouselMainDataType): JSX.Element {
+	const settings = {
+		dots: singleCarousel,
+		// autoplay: singleCarousel,
```
Comment (ShakirHajabba) [inline src/components/common/CarouselComponent/Carousel.component.tsx:46]: @PriyaVeruva remove commented code.

File: src/pages/Features/Features.module.scss [hunk 5]
```diff
@@ -0,0 +1,38 @@
+@use "../../styles/config.scss" as *;
+
+.featuresContainer {
```
Comment (ShakirHajabba) [inline src/pages/Features/Features.module.scss:3]: @PriyaVeruva Try nesting the classes as per the .tsx file.

File: src/pages/Tools/Tools.module.scss [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+	@include mobile-small {
+		grid-template-columns: repeat(1, minmax(0, 1fr));
+		padding-inline: 4vh;
+	}
+}
+
+.buttonContentSection {
+	font-size: 24px;
+
+	// font-family: $futuraMedium;
```
Comment (ShakirHajabba) [inline src/pages/Tools/Tools.module.scss:77]: @PriyaVeruva remove commented code.

File: src/components/common/Card/Card.scss (hunk 1 above)
Comment (PriyaVeruva) [reply src/components/common/Card/Card.scss:24]: issue resolved

File: src/components/common/Card/Card.scss (hunk 2 above)
Comment (PriyaVeruva) [reply src/components/common/Card/Card.scss:215]: Issue resolved

File: src/pages/Features/Features.module.scss (hunk 5 above)
Comment (PriyaVeruva) [reply src/pages/Features/Features.module.scss:3]: issue resolved

File: src/pages/Tools/Tools.module.scss (hunk 6 above)
Comment (PriyaVeruva) [reply src/pages/Tools/Tools.module.scss:77]: issue resolved

**Review by ShakirHajabba — APPROVED** [review]

**Pull request merged**